rds-deployment-HA/README.md

```diff
@@ -1,13 +1,24 @@
-# Provide High Availability to RDG and RDWA Server on top of Remote Desktop Sesson Collection deployment
+# Provide High Availability to RDG and RDWA Server on top of Remote Desktop Session Collection deployment
 
 This template deploys the following resources:
 
 <ul><li>a number of RD Gateway/RD Web Access vm (number defined by 'numberOfWebGwInstances' parameter)</li></ul>
 
-The template will join all new vms to the domain.
-Deploy RDS roles in the deployment.
-Join new VM's to the exisitng web and Gateway farm of basic RDS deplyment.
-Post configurations for web/Gateway VM's such as defining the Machine keys for IIS modules.
+The template will join all new VM’s to the domain.
+•	Deploy RDS roles in the deployment.
+•	Join new VM's to the existing web and Gateway farm of basic RDS deployment.
+•	Post configurations for web/Gateway VM's such as defining the Machine keys for IIS modules.
+
+Prerequisites:
+RDS-deployment-HA is an extension to the Basic-RDS-Deployment and it is mandatory to deploy any one of the template as prerequisite “rds-deployment”, “rds-deployment-custom-image-rdsh”, “rds-deployment-existing-ad”
```

Comment:
It's best to link these templates here so the user doesn't have to go find these. Please either link to Github location or Azure.com location.